linux_os/guide/system/permissions/partitions/mount_option_boot_efi_nosuid/ansible/shared.yml

```diff
@@ -0,0 +1,55 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = high
+- name: Check if system is UEFI
+  stat:
+    path: "/sys/firmware/efi"
+  register: system_uefi
+
+- name: 'Add nosuid Option to /boot/efi: Check information associated to mountpoint'
+  command: findmnt  '/boot/efi'
+  register: device_name
+  failed_when: device_name.rc > 1
+  changed_when: false
+
+- name: 'Add nosuid Option to /boot/efi: Create mount_info dictionary variable'
```

Comment:
How about adding all tasks below to a block? As they share the when condition `system_uefi.stat.exists`?

Reply:
will do